lib/wrappers/python/contourwall.py

```diff
@@ -91,7 +86,7 @@ def show(self, sleep_ms:int=0, optimize:bool=True):
         cw.show()
         """
         
-        ptr = ctypes.cast(self.pixels.tobytes(), ctypes.POINTER(ctypes.c_uint8))
+        ptr = (ctypes.c_uint8 * self.pixels.size).from_buffer(self.pixels)
```

Comment:
This variable declaration does not have a type, while it should have one.

Also if you run MyPy there is still an error on this line. If you change this line to this 

```python
ptr = ctypes.cast(ctypes.c_char_p(self.pixels.tobytes()), ctypes.POINTER(ctypes.c_uint8))
```

MyPy the types are correct.

The only thing you need to figure out is what type `ptr` is.